### PR TITLE
Add Python example

### PR DESCRIPTION
**Pull request by NerdCloudDevsPT**
## Related tickets & documents

* https://inoob.atlassian.net/browse/PP6-3

## Description

Add Python example code

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -626,6 +626,134 @@ public class DemoApplication {
...
+    self.stamp = stamp
+    self.reference = reference
+    self.amount = amount
+    self.currency = currency
+    self.language = language
+    self.items = items
+    self.customer = customer
+    self.redirectUrls = redirectUrls
+
+  # create toDictionary() method to customize a convertter of object class to dict type
```
Comment (villev1) [inline docs/examples.md:674]: Typo here, convertter -> converter

File: docs/examples.md [hunk 2]
```diff
@@ -626,6 +626,134 @@ public class DemoApplication {
...
+items = {item}
+customer = Customer("test.customer@example.com")
+redirectUrls = RedirectUrls("https://ecom.example.com/cart/success", "https://ecom.example.com/cart/cancel")
+
+b = Body(stamp, reference, amount, currency, language, items, customer,redirectUrls)
+
+body = json.dumps(b.toDictionary())
+encData = Crypto.calculate_hmac(Crypto,secret,headers,body)
+print("Encrypted data: " + encData)
+# result after run the program:
```
Comment (villev1) [inline docs/examples.md:752]: This comment should probably be "running the program"

File: docs/examples.md [hunk 3]
```diff
@@ -626,6 +626,134 @@ public class DemoApplication {
...
+    return hash.hexdigest()
+
+  #/
+  # @param secret Merchant shared secret
+  # @param hParams Headers or query string parameters
+  # @param body Request body or empty string for GET request
+  # @return
+  #/
+  @staticmethod
+  def calculate_hmac(self, secret: str, hparams: dict, body: str='') -> str:
```
Comment (villev1) [inline docs/examples.md:712]: hparams could be more informative, headerParams or similar. Also in comment hparams -variable is camel case but in code it is lowercase

File: docs/examples.md [hunk 4]
```diff
@@ -626,6 +626,134 @@ public class DemoApplication {
...
+
+  #/
+  # @param message Raw string
+  # @param secret Merchant shared secret
+  # @return
+  #/
+  @staticmethod
+  def compute_sha256_hash(message: str, secret: str) -> str:
+
+    # message must remove the whitespaces that were created during json parsing process
```
Comment (villev1) [inline docs/examples.md:701]: This a little weird comment as it says that message itself removes the whitespaces. Something like "whitespaces that were created during json parsing process must be removed" would make more sense.

**Review by villev1 — CHANGES_REQUESTED** [review]
Code looks good and works. Added few comments regarding style issues

**Review by loueranta-paytrail — APPROVED** [review]

**Pull request merged**